pkg/cloudprovider/providers/openstack/openstack_routes.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
This will fail the boilerplate header check:
run "hack/verify-boilerplate.sh"
" All rights reserved" should be removed from this line